appinventor/components/src/com/google/appinventor/components/runtime/Map.java

```diff
@@ -350,6 +353,29 @@ public void MapTypeAbstract(MapType type) {
     mapController.setMapTypeAbstract(type);
   }
 
+  /**
+   * @return Returns the custom URL of the base tile layer in use by the map.
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_TEXT,
+      defaultValue = "https://tile.openstreetmap.org/{z}/{x}/{y}.png")
```

Comment:
Hmm. So I have been going back and forth on this in my head. On the one hand, having a working URL here is a sensible default if someone switches to the Custom map type. However, if we always want people to provide a URL then making it the empty string would be the better option. Right now I'm still leaning to the former but it may not be clear to folks they need to change this.